### DISCUSSION TITLE
New Alerts extension on GitHub Markdown - similar to Admotions

### DISCUSSION THREAD

**Discussion by dellagustin-sap** [squidfunk/mkdocs-material#7035] [General]
Hello all,

I have stumbled upon a new feature on GitHub Markdown today, Alerts: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts

Looks a lot like Admotions, but the mapping is not 1:1 in terms of "types" they support.

I wanted to share this here to spark a discussion about having support for it on MkDocs Material, and see what would be needed for that. I'm not likely to contribute that, due to lack of capacity and familiarity with the development workflow.

**Comment by facelessuser** [comment]
Support already exists. Just install and configure https://github.com/oprypin/markdown-callouts. The output is the same as admonitions, so Material styles them just fine.

**Reply by dellagustin-sap** [reply]
Thank you. Is it compatible with the syntax used on GitHub? Based on the documentation it seems to be different.

The syntax on GitHub is:
```
> [!NOTE]
> Useful information that users should know, even when skimming content.
```

The syntax in the documentation of the extension you shared is:
```
> EXAMPLE: Hello world!
>
> * Item 1
> * Item 2
>
> Still going...
```

**Reply by facelessuser** [reply]
See the recent most release: https://github.com/oprypin/markdown-callouts/releases/tag/v0.4.0.

It isn't documented well, but it is there.

**Reply by dellagustin-sap** [reply]
That's great, thanks!
